sdk-http-apache-async/src/main/java/io/sphere/sdk/http/ApacheHttpClientAdapterImpl.java

```diff
@@ -97,7 +99,10 @@ private HttpUriRequest toApacheRequest(final HttpRequest httpRequest) throws Uns
             final HttpRequestBody body = httpRequest.getBody();
             final HttpEntity httpEntity;
             if (body instanceof StringHttpRequestBody) {
-                httpEntity = stringEntityOf(((StringHttpRequestBody) body).getString());
+                StringEntity stringEntity = new StringEntity(((StringHttpRequestBody) body).getString(),"UTF-8");
```

Comment:
We could make this vraible `final`. And instead of `UTF-8` we could use `java.nio.charset.StandardCharsets.UTF_8.name()`